src/firefly/java/edu/caltech/ipac/firefly/server/rpc/ResolveServerCommands.java

```diff
@@ -37,7 +41,40 @@ public String doCommand(SrvParam sp) throws Exception {
             }
         }
 
-        public boolean getCanCreateJson() { return false; }
+        public boolean getCanCreateJson() { return true; }
```

Comment:
This is why TargetPanel stopped working.

Reply:
Done, thanks!